README.md

```diff
@@ -28,11 +28,12 @@ const checkoutInstance = startCheckout(manifest, [options])
 
 **`manifest`** - server to server response from checkout session creation
 
-| Option      | Type       | Required | Default            | Description                                                                                  |
-| ----------- | ---------- | -------- | ------------------ | -------------------------------------------------------------------------------------------- |
-| `id`        | `string`   | no       | `easypay-checkout` | parameter to link the element in the page                                                    |
-| `onMessage` | `function` | no       | `() => {}`         | callback function to receive the checkout event. (for now, only `'complete'` is available)   |
-| `testing`   | `boolean`  | no       | `false`            | parameter to specify whether to use the testing API (`true`) or the production one (`false`) |
+| Option      | Type       | Required | Default            | Description                                                                   |
+| ----------- | ---------- | -------- | ------------------ | ----------------------------------------------------------------------------- |
+| `id`        | `string`   | no       | `easypay-checkout` | The id of the HTML element where the Checkout form should be included.        |
+| `onSuccess` | `function` | no       | `() => {}`         | Callback function to be called when the Checkout is finished succesfully.     |
```

Comment:
I assume you meant successfully finished. It's correct either way, but I like it better as written.

Reply:
The difference is in the first adverb, which is the one to enforce. So if the intention is to focus on the success state we should use it first, instead of finished.

Reply:
I feel the exact opposite, actually, I guess it's subjective. I agree with https://dictionary.cambridge.org/grammar/british-grammar/adverbs-and-adverb-phrases-position#:~:text=They%20usually%20go,is%20very%20long. that it loses emphasis in the middle of the sentence.

Reply:
They are not in the middle, they are at the end. But this is preference. For me is fine as u want.